__tests__/index.test.js

```diff
@@ -1,9 +1,47 @@
-// You can import your modules
-// const index = require('../index')
+const bot = require('..')
```

Comment:
Nit: can we call this `app`? We've been trying to avoid the word `bot` (I know, silly considering the framework is called Probot)

Reply:
Can do!